src/routes/business/businesses.ts

```diff
@@ -133,6 +133,30 @@ export const showBusinesses = async (req, res) => {
   return res.status(200).send(mappedBusinesses);
 };
 
+export const BUSINESS_UPDATE = "Patch/Businesses/business_id";
+
+/**
+ * Checks if the model has setted previously a value given in the input.
+ * This is useful to check if a Solaris entity may be updated or not checking the full entity
+ * or the desired part of the entity.
+ * i.e isChangeRequestRequired(mydata, business) or isChangeRequestRequired(mydata, business.address)
+ */
+const isChangeRequestRequired = (input, model) => {
+  let flag = false;
```

Comment:
```suggestion
  let isChangeRequired = false;
```
Could improve readability